docs/man/git-lfs-pointer.1.ronn

```diff
@@ -25,6 +26,11 @@ between different Git LFS implementations.
     Reads the pointer from STDIN to compare with the pointer generated from
     `--file`.
 
+* `--check`:
```

Comment:
Should we mention here that this flag has to be paired with either `--file` or `--stdin`? And that it's incompatible with `--pointer`?

Reply:
That's a great idea, and I appreciate your 👀 which are certainly much more careful than mine ;-).